### DISCUSSION TITLE
Try to compile loxilb follow document and found out iproute2 repo not found.

### DISCUSSION THREAD

**Discussion by broxio** [loxilb-io/loxilb#865] [Q&A] [answered]
I try to compile loxlb and try to run in on standalone mode, then I follow the document 

```bash
#!/bin/bash
# usage : curl -sfL https://github.com/loxilb-io/loxilb-tools/raw/main/loader/install.sh | sh -
git clone --recurse-submodules https://github.com/loxilb-io/iproute2.git && cd iproute2 && \
 cd libbpf/src/ && mkdir build && DESTDIR=build OBJDIR=build make install && cd - && \
 export PKG_CONFIG_PATH=$PKG_CONFIG_PATH:`pwd`/libbpf/src/ && \
 LIBBPF_FORCE=on LIBBPF_DIR=`pwd`/libbpf/src/build ./configure && make && \
 sudo cp -f tc/tc /usr/local/sbin/ntc && cd ../ && rm -fr iproute2
```

and I found out https://github.com/loxilb-io/iproute2.git has been took out from public access.

please kindly advise.

**Comment by TrekkieCoder** [comment, marked as answer]
Hi @broxio 

Sorry for the inconvenience. iproute2 is not used by loxilb anymore. So you can simply skip the step ```curl -sfL https://github.com/loxilb-io/tools/raw/main/loader/install.sh | sh -```.   Will update the build guide regarding the same.

**Reply by broxio** [reply]
Thank you for quick response, will skip this part,

after try to wander around Dockerfile ,I think I could make it works, Thanks for great pieces of software.
